lib/rules/disallow-spaces-inside-function-calls.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Disallows space after opening and before closing function call parentheses.
+ *
+ * Types: `Boolean` or `Object`
+ *
+ * Values:
+ * - true
+ * - `Object`:
+ *      - "only": An array containing tokens to disallow.
```

Comment:
I'd say we don't need too much options. Can we left just 2 params: `allExcept` with tokens array and, let's say, `arity` with 2 variants: single, multiple or even a Number.